### DISCUSSION TITLE
Suggestion: More alien tech should "leak" into human space.

### DISCUSSION THREAD

**Discussion by DoublePants** [endless-sky/endless-sky#10167] [Ideas]
One of the most fun things that happened during my first playthrough was when the Free Worlds chain has you encounter a ship with alien tech mods on it before I had actually found the wormhole that effectively allows infinite access to Hai tech (and Remnant tech). I'd like to suggest that there be more chances to collect bits of alien tech without actually encountering the wormhole or aliens in the first place.

It would provide a feeling of slower progression that I think the game desperately needs right now; currently I count essentially three tech hurdles:

1. Finding the wormhole
2. Finding the Remnant
3. Getting a Bactrian to capture your first Korath ship.

Adding smaller tech leaks into human space would add an effectively infinite number of smaller tech hurdles you can jump over before finding the wormhole, and would make no wormhole playthroughs a lot more satisfying.

I'd suggest the following possible methods of leaks:

- A small chance of a Spaceport encounter where an auction for a piece of alien tech is taking place.
- A very small chance of any ship spawning with a piece of alien tech installed (suddenly outfit scanners are useful!). [EDIT: I literally encountered this situation for the first time about 30 minutes after I made this post.]
- A small chance of finding a "crashed alien ship" on an uninhabited planet (one roll per planet per year, to prevent repeatedly landing on the same planet) which allows you to salvage a few functional outfits (but not the ship).
- A tiny chance of finding a "crashed alien ship" on an uninhabited planet (same as above) which still works and can be salvaged as a derelict. This would always be a smaller "civilian" ship, never a warship (no Korath ships except for 'olofez).
- This is a much bigger proposition, but the addition of a "black market" would also be a good way of giving players access to not only alien tech, but also a selection of military tech.

I think this would help the progression feel a lot more incremental than it does right now, rather than the current small, big jumps.

**Comment by DoublePants** [comment]
Side-note: I encountered my very first Marauder ship with alien tech on it literally 30 minutes after I made this post.

**Comment by Saugia** [comment]
As you've noted, there are a few examples of this happening. We generally keep this to mission or main campaign content though; each area in the game is meant to be a slow introduction into newer technology, and for example, damage types. We have the Hai tech that can occasionally be found on ships, and more importantly, in the main campaign, is also found on some special ships. Further along, you find that Korath tech has fallen into the hands of the Alphas. Or, the rare Korath spawns that can hop into some human space systems.

These are meant to be points in the story and not so much parts of the base setting of the game; for example, we wouldn't want generic human fleets or ships (no matter how rare) flying around with remnant, korath, or coalition technology. I don't think what you say are "big jumps" are really all that big; most of it in the main story leads up to one point or another; there's nothing metric-wise that you could explain as "big vs small" in a case where it's you either see the tech or you don't (and again, missions/campaign do have those small leaks).

**Reply by DoublePants** [reply]
Ah, by "big jumps" I mean the player going from "has no meaningful access to a tech category" to "has complete access to a tech category." For example, once you get your first license from the Remnant, you go from having basically no access to Remnant tech to having almost complete access to it with the exception of a couple of their ship types. Once you find the wormhole, you go from having basically no access to Hai tech to having almost complete access.

That is, a "big jump" is "suddenly gaining a large amount of access," whereas a small jump would be something like "you now have access to a single Hai engine."

**Comment by TheGiraffe3** [comment]
In response to 3, a Kimek Spire is less than half the Bactrian’s cost, although it has less powerful weapons. 
But yes, I think this is a great idea. Unfortunately (except for the Hai) there is no good lore reason for that-if you get something alien, you want to keep it!

**Reply by DoublePants** [reply]
I mean, if you're a player playing the game, sure. But if you were an actual person in the universe, is it more valuable to you to hold onto some random piece of alien tech that might not even be useful to you, or to auction it off for millions of dollars?

I mean, if you personally stumbled across an alien device used for cooling their starships, would you just keep it if you could sell it and become rich?

**Reply by TheGiraffe3** [reply]
Cooling system, I’d sell. But if I find this fancy alien weapon that heats people up (Fire-Lance), I’m not going to sell it.

**Reply by DoublePants** [reply]
Why not? Are you going to build a space ship and mount it on there and go kill someone with it?

I'd totally sell it to some nation's military for billions of dollars.

**Reply by TheGiraffe3** [reply]
Well, I could be flying a Star Queen (or Bactrian) and be going and capturing Falcons for also millions of credits. This way, it’s easier to disable the enemy ship. (or in a Bactrian, go fight the Korath and overheat them. hmm…)

**Reply by DoublePants** [reply]
Most people aren't willing to risk death to go fight pirates. That's what main characters in movies do, not normal humans.

**Reply by TheGiraffe3** [reply]
I suppose the player isn’t a normal human, they can reload saves…
But if you’re in a Bactrian, a Falcon’s not going to be able to destroy you. You will disable and capture it, and hey presto. Three million credits for just a few days.

**Reply by lumbar527** [reply]
> Most people aren't willing to risk death to go fight pirates. That's what main characters in movies do, not normal humans.

I like going around fighting pirates in whatever warship I have. <first> <last> is very much not a normal human.

**Comment by yafflehk** [comment]
I always thought that the syndicate storyline should be triggered by the player buying a fancy new reactor from the syndicate that then blows up, leaving they player stranded and penniless (fined by the local spaceport for having a ship blow up in the dock). You then have to build your character back up while investigating the syndicate's shady business practices, leading to the Korath and a jump drives etc.
